tests/requirements.txt

```diff
@@ -1,3 +1,3 @@
 conda-content-trust
-rich
```

Comment:
Huh, this got dropped?

Reply:
Yup,  this was needed for `_reposerver.py` but not anymore.